packages/slice-machine/src/modules/toaster/index.tsx

```diff
@@ -9,6 +9,7 @@ export enum ToasterType {
   ERROR = "error",
   INFO = "info",
   LOADING = "loading",
+  SCREENSHOT_FAILED = "screenshot_failed",
```

Comment:
I don't think we need this. We should just use the `ERROR` type instead. The only reason SCREENSHOT_CAPTURED was a new type is because the toaster looked different

Reply:
I had no clue haha, thanks